storage/allocator.go

```diff
@@ -41,6 +42,10 @@ const (
 	// is within rebalanceFromMean of the mean, it is considered a
 	// viable target to rebalance to.
 	rebalanceFromMean = 0.025 // 2.5%
+	// rebalanceShouldRebalanceChance represents a chance that an individual
```

Comment:
Scratch that "insanely high" comment; I misunderstood how this was being used. I thought it was a floor to reshuffle things on a mostly-stable cluster, not a ceiling to decide which replicas get moved. Skipping 95% of replicas in `shouldRebalance` seems like an indirect way to choose a random replica for rebalancing; it's hard to reason about what effect changing this parameter will have. I feel like "store decides to rebalance and then chooses a random replica" is a more understandable model than checking the store-level `shouldRebalance` property in a replica-level loop.